apps/web/playwright/impersonation.e2e.ts

```diff
@@ -5,7 +5,7 @@ import { test } from "./lib/fixtures";
 test.describe.configure({ mode: "parallel" });
 
 test.describe("Users can impersonate", async () => {
-  test.afterAll(async ({ users }) => {
+  test.afterEach(async ({ users }) => {
```

Comment:
Seems like afterAll doesn't have fixtures any more

Reply:
Before it was more a 'bug' that it had access to 'test fixtures'. 'test fixtures' -> fixtures which get created for every test, they should be used in {before,after}Each.